app/tests/spec/views/sign_up.js

```diff
@@ -272,6 +273,7 @@ define(function (require, exports, module) {
 
         return view.render()
           .then(function () {
+            assert.lengthOf(view.$('#suggest-sync'), 0);
             assert.equal(view.$('.info.nudge').html(), 'Migrate your sync data by creating a new Firefox&nbsp;Account.');
```

Comment:
Shouldnt this be some `id` too? It feels inconsistent compared to the other assert.equal(`selector`).html() calls.